lib/paramedic.js

```diff
@@ -393,7 +393,7 @@ class ParamedicRunner {
         return Q().then(() => paramedicTargetChooser.chooseTarget(/* useEmulator= */true, /* preferredTarget= */this.config.getTarget()))
             .then((targetObj) => {
                 this.targetObj = targetObj;
-                cmd += ' --target ' + this.targetObj.target;
+                cmd += ` --target "${this.targetObj.target}"`;
```

Comment:
Adding the `"` here changes quite a lot how this works. Intended?

Reply:
Yes this was intended.

Here is an example case, from terminal, when the argument was not wrapped. Since the command is built in string and then executed, I wanted to make sure we handled it the same way as it would be typed in terminal.
```
$ cordova run --target iPhone-6s, 12.2 ios
The platform "12.2" does not appear to have been added to this project.

$ cordova run ios --target iPhone-6s, 12.2
The platform "12.2" does not appear to have been added to this project.
```

Reply:
https://github.com/apache/cordova-paramedic/pull/21, which implements the same thing, has some more related changes that might be useful/necessary.